### PR TITLE
e2e fix: The kind image name is wrong.

### PR DESCRIPTION
**Pull request by ty-dc**
#### What type of PR is this?

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ty-dc** [comment]
E2E_KIND_IMAGE_TAG 写错为E2E_KIND_NODE_IMAGE

**Review by Icarus9913 — APPROVED** [review]

File: test/Makefile [hunk 1]
```diff
@@ -155,7 +155,7 @@ setup_kind:
   		fi
 	$(QUIET) cat $(CLUSTER_DIR)/$(E2E_CLUSTER_NAME)/kind-config.yaml
 	KIND_OPTION="" ; \
-       		[ -n "$(E2E_KIND_NODE_IMAGE)" ] && KIND_OPTION=" --image $(E2E_KIND_NODE_IMAGE) " && echo "setup kind with E2E_KIND_NODE_IMAGE=$(E2E_KIND_NODE_IMAGE)"; \
```
Comment (weizhoublue) [inline test/Makefile:158]: 原来的 变量是否 
if E2E_CHINA_IMAGE_REGISTRY
      E2E_KIND_NODE_IMAGE:=xxxxx/kindest/node:$(E2E_KIND_IMAGE_TAG)
else
      E2E_KIND_NODE_IMAGE:=kindest/node:$(E2E_KIND_IMAGE_TAG)

Comment (ty-dc) [reply test/Makefile:158]: 本身有这样的变量，已修改，引用该变量。

**Review by Icarus9913 — APPROVED** [review]

**Review by weizhoublue — APPROVED** [review]

**Comment by weizhoublue** [comment]
我们是稳定维护多少个版本，结果只管 0.9 ？

**Comment by ty-dc** [comment]
> /hold

/un hold rebase main

**Pull request merged**